app/routes/questions.actions.tsx

```diff
@@ -173,13 +184,9 @@ export const Action = ({
       <input type="hidden" name="incBy" value={actionTaken ? -1 : 1} />
       <input type="hidden" name="stateString" value={stateString} />
       {children}
-      <Button className={className}>
+      <ButtonSecondary disabled={disabled} active={actionTaken} tooltip={hint}>
```

Comment:
this is also just a normal button with a custom tooltip

Reply:
yup